src/main/java/edu/umich/its/cpm/BoxUtils.java

```diff
@@ -515,7 +515,7 @@ private static String get_box_user_name(BoxUser.Info uInfo) {
 	public static String getBoxClientId(String userId) {
 		String boxClientId = env.getProperty(Utils.BOX_CLIENT_ID);
 		
-		if (Utils.isCurrentUserCPMAdmin(userId, env)) {
+		if (Utils.isCurrentUserCPMAdmin(userId, env) || env.getProperty(Utils.BOX_ADMIN_ACCOUNT_ID).equals(userId)) {
```

Comment:
Consider one method that gets both the id and secret.  That would ensure the security checks are the same.